src/lib/eZ/RichText/Converter/Render/Template.php

```diff
@@ -221,4 +223,32 @@ protected function extractTemplateConfiguration(DOMElement $template, DOMXPath $
 
         return $this->extractHash($configElements->item(0));
     }
+
+    /**
+     * BC: wrap nested content containing line breaks with "literallayout" DocBook tag,
+     * unless literallayout already exists.
+     *
+     * @param \DOMNode $rootNode
+     * @param \DOMNode $node
+     *
+     * @return \DOMNode
+     */
+    private function wrapContentWithLiteralLayout(DOMNode $rootNode, DOMNode $node): DOMNode
+    {
+        $xpath = new DOMXPath($node->ownerDocument);
+        $xpath->registerNamespace('docbook', 'http://docbook.org/ns/docbook');
+        if (
+            strpos($node->nodeValue, "\n") !== false
```

Comment:
```suggestion
            strpos($node->nodeValue, PHP_EOL) !== false
```

Reply:
> ```diff
> +          strpos($node->nodeValue, "\n") !== false
> -          strpos($node->nodeValue, PHP_EOL) !== false
> ```
I would disagree with that one. This is not `PHP_EOL`, this is [DocBook "\n"](https://github.com/ezsystems/ezplatform-richtext/blob/6cdb5054c6ee2eda28d0f10a8a2b45b2f6b50a89/src/lib/eZ/RichText/Resources/stylesheets/docbook/xhtml5/edit/core.xsl#L93) which accidentally is the same.
Maybe it calls for dedicated const, not sure however in which class to put it.